lib/main.dart

```diff
@@ -116,7 +117,7 @@ class MyApp extends StatelessWidget {
           routes: [
             GoRoute(
                 path: 'play',
-                pageBuilder: (context, state) => buildTransition(
+                pageBuilder: (context, state) => buildTransition<void>(
```

Comment:
Is this needed now?

Reply:
Assuming the purpose of this repo is to show how to use game_template, an explicit <void> may help developers understand that a custom type can be used in Page<T>. This can be taken out though, as it is unnecessary code for this repo